.github/workflows/pr-merged.yaml

```diff
@@ -9,15 +9,16 @@ on:
       - '*.md'
       - '*.yaml'
       - '.github/workflows/*'
+  push:
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
 jobs:
   pr-merged:
-    if: ${{ github.event.pull_request.merged == true }} ## Skip if not merged
-    uses: kubescape/workflows/.github/workflows/incluster-comp-pr-merged.yaml@main
+    #if: ${{ github.event.pull_request.merged == true }} ## Skip if not merged
```

Comment:
@alegrey91  Why did you remove this statement?

Reply:
Testing purposes. I'll remove this once the workflow works.